cypress/support/awx-commands.ts

```diff
@@ -483,7 +480,7 @@ Cypress.Commands.add('selectDetailsPageKebabAction', (dataCy: string) => {
   cy.get('[data-cy="actions-dropdown"]')
     .click()
     .then(() => {
-      cy.getByDataCy(dataCy).click();
+      cy.get(`[data-cy="${dataCy}"]`).click();
```

Comment:
Sometimes actions in the kebab are disabled and we need to use the getBy to wait until enabled before clicking